babel.config.js

```diff
@@ -1,96 +1,2 @@
-         */
-        development: {
-            // Ignore everything with underscores except stories
-            ignore: [/\/__(tests?|mocks|fixtures|helpers|dist)__\//],
-            plugins: [
-                ...plugins,
-                [
-                    'module-resolver',
-                    {
-                        root: ['./'],
-                        /**
-                         * Exported modules will be consumed by other projects
-                         * which import Venia. Those projects will need to
-                         * override the 'src/drivers' dependency so Venia
-                         * modules will run outside the Venia app. This alias
-                         * exports the modules so the drivers dependency is
-                         * a unique string '@magento/venia-drivers', which is
-                         * less likely to collide with an existing dependency
-                         * than 'src/drivers' is.
-                         *
-                         * In webpack (or any build system) config for a project
-                         * using Venia modules, you must write an override for
-                         * '@magento/venia-drivers' and make an alias to that
-                         * module in your build configuration, e.g.:
-                         *
-                         *     alias: {
-                         *       '@magento/venia-drivers': './src/veniaDrivers'
-                         *     }
-                         *
-                         * to map from this virtual string to your replacement.
-                         */
-                        alias: {
-                            '^src/drivers$': '@magento/venia-drivers'
-                        },
-                        /**
-                         * Suppress console warning about missing dependencies.
-                         */
-                        loglevel: 'silent'
-                    }
-                ]
-            ],
-            presets: [
-                ['@babel/preset-env', { modules: false, targets: targets.dev }]
-            ]
-        },
-        production: {
-            plugins: [
-                ...plugins,
-                [
-                    '@babel/plugin-transform-runtime',
-                    { helpers: true, regenerator: true }
-                ]
-            ],
-            presets: [
-                ['@babel/preset-env', { modules: false, targets: targets.prod }]
-            ]
-        },
-        test: {
-            plugins: [...plugins, ['babel-plugin-dynamic-import-node']],
-            presets: [
-                [
-                    '@babel/preset-env',
-                    { modules: 'commonjs', targets: targets.test }
-                ]
-            ]
-        }
-    };
-    return envConfigs[api.env() || 'development'];
-};
-
-module.exports = config;
+// Venia owns code style, and the rest of the project shares it.
```

Comment:
Well... it will after this change :D Peregrine also uses this, so why not have this live in Peregrine? Maybe we should just keep this, prettier config, etc in a top-level `config/` directory and import them specifically within the sub-modules/packages.

Reply:
Probably a better option is to make a babel preset. For now, I can switch this so it copies the babel config before publish.

Reply:
@sirugh I reverted this change and I'll approach it from another angle in another PR.